src/highlightingmappings.h

```diff
@@ -1438,29 +1438,29 @@ static const HLStyle highlighting_styles_TXT2TAGS[] =
 	{ SCE_TXT2TAGS_DEFAULT,		"default",		FALSE },
 	{ SCE_TXT2TAGS_LINE_BEGIN,	"default",		FALSE }, /* XIFME: remappings should be avoided */
 	{ SCE_TXT2TAGS_PRECHAR,		"default",		FALSE },
-	{ SCE_TXT2TAGS_STRONG1,		"strong",		FALSE },
-	{ SCE_TXT2TAGS_STRONG2,		"strong",		FALSE },
-	{ SCE_TXT2TAGS_EM1,			"emphasis",		FALSE },
-	{ SCE_TXT2TAGS_EM2,			"underlined",	FALSE }, /* WTF? */
-	{ SCE_TXT2TAGS_HEADER1,		"header1",		FALSE },
-	{ SCE_TXT2TAGS_HEADER2,		"header2",		FALSE },
-	{ SCE_TXT2TAGS_HEADER3,		"header3",		FALSE },
-	{ SCE_TXT2TAGS_HEADER4,		"header4",		FALSE },
-	{ SCE_TXT2TAGS_HEADER5,		"header5",		FALSE },
-	{ SCE_TXT2TAGS_HEADER6,		"header6",		FALSE },
-	{ SCE_TXT2TAGS_ULIST_ITEM,	"ulist_item",	FALSE },
-	{ SCE_TXT2TAGS_OLIST_ITEM,	"olist_item",	FALSE },
-	{ SCE_TXT2TAGS_BLOCKQUOTE,	"blockquote",	FALSE },
-	{ SCE_TXT2TAGS_STRIKEOUT,	"strikeout",	FALSE },
-	{ SCE_TXT2TAGS_HRULE,		"hrule",		FALSE },
-	{ SCE_TXT2TAGS_LINK,		"link",			FALSE },
-	{ SCE_TXT2TAGS_CODE,		"code",			FALSE },
-	{ SCE_TXT2TAGS_CODE2,		"code",			FALSE },
-	{ SCE_TXT2TAGS_CODEBK,		"codebk",		FALSE },
-	{ SCE_TXT2TAGS_COMMENT,		"comment",		FALSE },
-	{ SCE_TXT2TAGS_OPTION,		"option",		FALSE },
-	{ SCE_TXT2TAGS_PREPROC,		"preproc",		FALSE },
-	{ SCE_TXT2TAGS_POSTPROC,	"postproc",		FALSE }
+	{ SCE_TXT2TAGS_STRONG1,		"markup_strong",		FALSE },
+	{ SCE_TXT2TAGS_STRONG2,		"markup_strong",		FALSE },
+	{ SCE_TXT2TAGS_EM1,			"markup_emphasis",		FALSE },
+	{ SCE_TXT2TAGS_EM2,			"markup_underlined",	FALSE }, /* WTF? */
+	{ SCE_TXT2TAGS_HEADER1,		"markup_header_1",		FALSE },
+	{ SCE_TXT2TAGS_HEADER2,		"markup_header_2",		FALSE },
+	{ SCE_TXT2TAGS_HEADER3,		"markup_header_3",		FALSE },
+	{ SCE_TXT2TAGS_HEADER4,		"markup_header_4",		FALSE },
+	{ SCE_TXT2TAGS_HEADER5,		"markup_header_5",		FALSE },
+	{ SCE_TXT2TAGS_HEADER6,		"markup_header_6",		FALSE },
+	{ SCE_TXT2TAGS_ULIST_ITEM,	"markup_ulist_item",	FALSE },
+	{ SCE_TXT2TAGS_OLIST_ITEM,	"markup_olist_item",	FALSE },
+	{ SCE_TXT2TAGS_BLOCKQUOTE,	"markup_blockquote",	FALSE },
+	{ SCE_TXT2TAGS_STRIKEOUT,	"markup_strikeout",	FALSE },
+	{ SCE_TXT2TAGS_HRULE,		"markup_hrule",		FALSE },
+	{ SCE_TXT2TAGS_LINK,		"markup_link",			FALSE },
+	{ SCE_TXT2TAGS_CODE,		"markup_code",			FALSE },
+	{ SCE_TXT2TAGS_CODE2,		"markup_code",			FALSE },
+	{ SCE_TXT2TAGS_CODEBK,		"markup_code_block",		FALSE },
+	{ SCE_TXT2TAGS_COMMENT,		"markup_comment",		FALSE },
+	{ SCE_TXT2TAGS_OPTION,		"markup_option",		FALSE },
+	{ SCE_TXT2TAGS_PREPROC,		"markup_preproc",		FALSE },
+	{ SCE_TXT2TAGS_POSTPROC,	"markup_postproc",		FALSE }
```

Comment:
no need to touch these names, they are specific to the Txt2tags filetype no matter what.  It's only the name used as the key in the *data/filetypes.txt2tags*, it doesn't have to match the named style.